src/refactor_nrepl/ns/tracker.clj

```diff
@@ -60,31 +60,38 @@
                      (deps-set ns))]
       file)))
 
-(defn- in-refresh-dirs? [refresh-dirs file]
+(defn- absolutize-refresh-dirs [refresh-dirs]
```

Comment:
Won't it be better to have a generic helper function operating on dirs and just apply it to `refresh-dirs`? Seems cleaner to me.